lib/v2/kloteneranzeiger/radar.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    'kloteneranzeiger.ch': {
+        _name: 'Klotener Anzeiger',
+        '.': [
+            {
+                title: 'Klotener Anzeiger New',
+                docs: 'https://docs.rsshub.app/routes/blog#klotener-anzeiger',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/93d9bb53ab6ba75f036f7b563b0ec10bd45fdc08/website/docs/routes/traditional-media.mdx#L305

The docs you've entered is `traditional-media` not `blog`